src/locales/es.ts

```diff
@@ -59,247 +62,236 @@ export default {
+    mary_soledad_torres_acosta_virgin: 'Santa María Soledad Torres Acosta, Vírgen',
+    perpetua_of_carthage_and_felicity_of_carthage_martyrs: 'Santas Perpetua y Felicidad, Mártires',
+    philip_and_james_apostles: 'Santos Felipe y Santiago, Apóstoles',
+    roch_gonzalez_alphonsus_rodriguez_and_john_del_castillo_priests: 'Santos Roque González, Alfonso Rodríguez Olmedo, y Juan del Castillo, Sacerdotes y Mártires',
+    simon_and_jude_apostles: 'Santos Simón y Judas, Apóstoles',
+    stephen_the_first_martyr: 'San Esteban, Primer Mártir',
+    teresa_benedicta_of_the_cross_stein_virgin: 'Santa Teresa Benedicta de la Cruz (Edith Stein), Vírgen y Mártir',
+    teresa_of_calcutta_religious: 'Santa Teresa de Calcuta, Religiosa',
+    teresa_of_jesus_of_avila_virgin: 'Santa Teresa de Jesús, Vírgen y Doctor',  // TODO: Localise as `Saint Teresa of Jesus of Ávila, Virgin and Doctor of the Church`
+    teresa_of_jesus_of_los_andes_virgin: 'Santa Teresa de Los Andes, Vírgen',  // TODO: Localise as `Saint Teresa of Jesus of Los Andes, Virgin`
+    therese_of_the_child_jesus_and_the_holy_face_of_lisieux_virgin: 'Santa Teresita del Niño Jesús, Vírgen y Doctor',
+    therese_of_the_child_jesus_and_the_holy_face_of_lisieux_virgin_secondary_patroness_of_france: 'Santa Teresita del Niño Jesús, Vírgen, Patrona Secundaria de Francia',
+    thomas_aquinas_priest: 'Santo Tomás de Aquino, Obispo y Doctor',
+    thomas_apostle: 'Santo Tomás, Apóstol',
+    vincent_de_paul_priest: 'San Vicente de Paúl, Obispo',
+    holy_child_of_cebu: 'Divino Niño',  // TODO: Should it not be rather `Santo Niño de Cebú`?
+    most_holy_name_of_jesus: 'Santísimo Nombre de Jesús',
+    vietnamese_martyrs: 'Mártires de Vietnam',
+    visitation_of_mary: 'Visitación de la Santísima Vírgen María',
+    our_lady_of_sorrows_principal_patroness_of_slovakia: 'Nuestra Señora del Socorro, Patrona Principal de Eslovaquia',
+    george_of_lydda_martyr_principal_patron_of_england: 'San Jorge, Mártir y Patrono Principal de Inglaterra',
+    teresa_benedicta_of_the_cross_stein_virgin_secondary_patron_of_europe: 'Santa Teresa Benedicta de la Cruz (Edith Stein), Vírgen, Mártir y Patrona Secundaria de Europa',
+    andrew_apostle_principal_patron_of_scotland: 'San Andrés, Apóstol y Patrono Principal de Escocia',
+    david_of_wales_bishop_principal_patron_of_wales: 'San David, Obispo y Patrono Principal de Gales',
+    catherine_of_siena_virgin_secundary_patroness_of_europe: 'Santa Catalina de Siena, Vírgen, Doctor de la Iglesia y Patrona Secundaria de Europa',
+    catherine_of_siena_virgin_secondary_patroness_of_italy_and_europe: 'Santa Catalina de Siena, Vírgen, Doctor de la Iglesia y Patrona Secundaria de Italia y Europa',
+    our_lady_of_guadalupe_principal_patroness_of_the_americas: 'Nuestra Señora de Guadalupe, Patrona Principal de América',
+    francis_of_assisi_principal_patron_of_italy: 'San Francisco of Asís, Patrono de Italia',
```

Comment:
@CodingAleCR, could you proof the Spanish patronage titles please? Below I list them all. There are also some other issue which you might be able to fix. Thanks in advance. :smiley:

```js
our_lady_of_aparecida_principal_patroness_of_brazil: 'Nuestra Señora de Aparecida, Patrona Principal de Brasil',
our_lady_of_lujan_principal_patroness_of_argentina: 'Nuestra Señora de Luján, Patrona Principal de Argentina',
joachim_and_anne_principal_patroness_of_the_province_of_quebec_parents_of_mary: 'Santa Ana, Patrona Principal de Quebec, y San Joaquín, Padres de la Santísima Vírgen María',
brigid_of_kildare_virgin_secondary_patron_of_ireland: 'Santa Brígida de Kildare, Vírgen y Patrona Secundaria de Irlanda',
james_apostle_principal_patron_of_spain: 'Santiago, Apóstol, Patrono Principal de España',
joseph_spouse_of_mary_principal_patron_of_canada: 'San José, Esposo de la Santísima Vírgen María, Patrono Principal de Canadá',
teresa_of_jesus_of_avila_virgin: 'Santa Teresa de Jesús, Vírgen y Doctor',  // TODO: Localise as `Saint Teresa of Jesus of Ávila, Virgin and Doctor of the Church`
teresa_of_jesus_of_los_andes_virgin: 'Santa Teresa de Los Andes, Vírgen',  // TODO: Localise as `Saint Teresa of Jesus of Los Andes, Virgin`
therese_of_the_child_jesus_and_the_holy_face_of_lisieux_virgin_secondary_patroness_of_france: 'Santa Teresita del Niño Jesús, Vírgen, Patrona Secundaria de Francia',
holy_child_of_cebu: 'Divino Niño',  // TODO: Should it not be rather `Santo Niño de Cebú`?
our_lady_of_sorrows_principal_patroness_of_slovakia: 'Nuestra Señora del Socorro, Patrona Principal de Eslovaquia',
george_of_lydda_martyr_principal_patron_of_england: 'San Jorge, Mártir y Patrono Principal de Inglaterra',
teresa_benedicta_of_the_cross_stein_virgin_secondary_patron_of_europe: 'Santa Teresa Benedicta de la Cruz (Edith Stein), Vírgen, Mártir y Patrona Secundaria de Europa',
andrew_apostle_principal_patron_of_scotland: 'San Andrés, Apóstol y Patrono Principal de Escocia',
david_of_wales_bishop_principal_patron_of_wales: 'San David, Obispo y Patrono Principal de Gales',
catherine_of_siena_virgin_secundary_patroness_of_europe: 'Santa Catalina de Siena, Vírgen, Doctor de la Iglesia y Patrona Secundaria de Europa',
catherine_of_siena_virgin_secondary_patroness_of_italy_and_europe: 'Santa Catalina de Siena, Vírgen, Doctor de la Iglesia y Patrona Secundaria de Italia y Europa',
our_lady_of_guadalupe_principal_patroness_of_the_americas: 'Nuestra Señora de Guadalupe, Patrona Principal de América',
francis_of_assisi_principal_patron_of_italy: 'San Francisco of Asís, Patrono de Italia',
```

Reply:
My only suggestion would be to use 'of Avila' and 'of Los Andes' since it's the more distinctive name, and the results you find while looking for that name are usually correct and avoid confusion between saints of the same name.

```
teresa_of_jesus_of_avila_virgin: 'Santa Teresa de Ávila, Vírgen y Doctora',  // TODO: Localise as `Saint Teresa of Jesus of Ávila, Virgin and Doctor of the Church`
teresa_of_jesus_of_los_andes_virgin: 'Santa Teresa de Los Andes, Vírgen',  // TODO: Localise as `Saint Teresa of Jesus of Los Andes, Virgin
```

With this particular one, the only celebrations I could find reference it as in the way you suggested in the todo, so I´d use it right away.

```
holy_child_of_cebu: 'Santo Niño de Cebú',
```

Other than those suggestions (which could very well stay the same way you translated them) you are definitely spot on. Nice spanish 😄!

Reply:
> My only suggestion would be to use 'of Avila' and 'of Los Andes' since it's the more distinctive name, and the results you find while looking for that name are usually correct and avoid confusion between saints of the same name.

Well, it might be a good idea to look in the official Spanish translation of the Roman Missal, how it names the saint; St Teresa of Jesus of Avila is celebrated world-wide, so you should that celebration of 15 October. In Slovakia, we celebrate her as _St Teresa of Jesus_ and her namesake of Lisieaux as _St Therese of Child Jesus_. IMO, even the names of these two saints are quiet ambiguous, bu there are many `St Teresias/Thereses of [Child] Jesus`, therefore I suggest to _always_ append either their surnames or the places they are connected with, while their religious names should be output in full, _**even if their name in the celebration name in the (vernacular) Roman Missal does not include it**_. What do you think, @emagnier?

> Nice spanish smile!

I don’t speak Spanish, though I understand a bit, but thanks! :smiley: I am capable of searching the Internet, esp using Google Translate! :smiley:

Reply:
I have found [iglesiaactualidad.wordpress.com](https://iglesiaactualidad.wordpress.com/liturgia/misal-romano-tercera-edicion/) website which contains links with portions (or whole?) from the Roman Missal in Spanish (editio typica tertia). Those links comes from https://textosparalaliturgia.blogspot.com, where I have found the [texts for the celebration of St Teresa of Jesus (of Ávila)](https://textosparalaliturgia.blogspot.com/2017/09/misal-romano-tercera-edicion-santa_6.html), which clearly says she is called `Santa Teresa de Jesús`, i.e. `Saint Teresa of Jesus`, which is ambiguous in a project like romcal where we may (and will) list many saints/blesseds of that very same name. BTW, the same name is used in Slovak for her and I think that the case is in every language.

@emagnier, I suggest to _always append_ the surname (or place or whatever that can be considered as a surname) to all religious names regardless if the surname is included in the Roman Missal or not. For example in this case, we should append `of Ávila` → `Saint Teresa of Jesus of Ávila`.

Reply:
BTW, same is true in the [Latin Roman Missal](https://drive.google.com/uc?export=view&id=12HSqEb71ysw6S4fzAIw04CmGW2N2vqUe#page=79) and in [English Roman Missal](https://drive.google.com/uc?export=view&id=12u_SI0RoL5BW2yFiEZBxbdgqHNGi8yzE#page=1441), too.

Reply:
@emagnier, I still need confirmation if my proposal (to append ‘surname’, e.g. `of Ávila` to all religious names in all locales regardless if it is printed in the official liturgical books or not) is okay with you. I think it _might_ cause some ambiguity if the ‘surname’ is not inserted, albeit not in a single calendar.

Reply:
For sure it's better to append "surname". 
Indeed the surname is generally the "localisation" from where a saint comes from. Especially the saints from the first centuries: Gregory of Nyssa, Gregory of Nazianzus...

Reply:
Thanks you agree with me. :smiley:

Reply:
@emagnier, it just got into my mind: what about `anthony_of_egypt_abbot` (and the like)? That is personal celebrations celebrated in `general.ts` that don’t use ‘surname’.

I see two possible solutions for such celebrations:
1. make them an exception and don’t append the ‘surname’:
   - reasoning: as they are inscribed into `general.ts`, all those who created the particular (local) calendars, must have already dealt with the problem of ambiguity;
2. append the ‘surname’ to all personal celebrations without exception:
   - reasoning: even though they are inscribed into `general.ts` and those who created the particular calendars have dealt with tthe problem of ambiguity, they might still cause some ambiguity (actually, they always caused me _personally_ this kind of ambiguity), because (e.g.) there are many St Anthonies and many St Teresias of Jesus and one needs to either know the differences (e.g. different titles or at least the date of celebration) or look for them somewhere.

Reply:
You like the rules :) Please only add surnames or any other adjectives (like localisation) only if the Blessed/Saint is well known by this full name, or only if there is a risk of confusion with another person. As much as possible things should be kept natural, or close than the name used in the missal. Here we already have `_of_egypt` and `_abbot` and it's enough ;)

Reply:
> You like the rules :)

I prefer having precisely defined rules to having no or imprecisely defined rules. :smiley:

> Please only add surnames or any other adjectives (like localisation) only if the Blessed/Saint is well known by this full name, or only if there is a risk of confusion with another person. As much as possible things should be kept natural, or close than the name used in the missal.

Okay.

> Here we already have `_of_egypt` and `_abbot` and it's enough ;)

Note that I talk **not** about the key _names_, but key _values_. All key names already always contain the ‘surnames’ and that is enough to distinguish them. Here I talk about the _values_, that is, e.g., should we use `Saint Anthony, Abbot` or `Saint Anthony of Egypt, Abbot`. However, I think you have already answered my question. :wink:

Reply:
> I prefer having precisely defined rules to having no or imprecisely defined rules. 😃

But the reason (or the rules) isn't always the right way to go, you also need to use discernment and good sense ;)

Reply:
I agree with you, however, I use my descernment and good sense in creation of rules that are good and make sense. And as romcal is a project where no one person rules, I always ask for input from other contributors (although usually only you answer). :smiley: